apps/wing/src/analytics/event.ts

```diff
@@ -0,0 +1,17 @@
+import { CIData } from "./collectors/ci-collector";
+import { CLIData } from "./collectors/cli-collector";
+import { NodeData } from "./collectors/node-collector";
+import { OSData } from "./collectors/os-collector";
+
+export interface AnalyticEvent {
+  event: string;
+  timestamp?: string;
+  anonymousId?: string;
+  properties: {
+    inCI: boolean;
```

Comment:
do we really need both `inCI` and `ci`. We can assume that if `ci` has contents we are in a CI context, no?

Reply:
I went back and forth on this. In my mind it would depend on what we are using downstream to query/visualize these analytics. I assume you are right, but I wonder if having this easy true/false field would make some query easier downstream. 🤷 Happy to take it out, I wonder if @ekeren has any thoughts here?